frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash';
+import { Humanize } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+
+const getTotal = (stats: StackDataPoint[]) => {
+  return stats.reduce((total, dataPoint) => total + dataPoint.y, 0);
+};
+
+const addOthers = (stats: StackDataPoint[], totalUsed, formatValue) => {
+  const top5Total = getTotal(stats);
+  const others = totalUsed - top5Total;
+  const othersData = {
+    x: '',
+    y: others,
+    name: 'Others',
+    color: 'black',
+    label: formatValue(others).string,
+  };
+  return othersData;
+};
+
+export const addAvailable = (stats: StackDataPoint[], total, used, totalUsed, formatValue) => {
+  const availableInBytes = Number(total) - Number(used);
+  let othersData = {};
+  if (stats.length === 5) {
+    othersData = addOthers(stats, totalUsed, formatValue);
+  }
+  const availableData = {
+    x: '',
+    y: availableInBytes,
+    label: `${formatValue(availableInBytes).string}`,
+  };
+  return _.isEmpty(othersData) ? [...stats, availableData] : [...stats, othersData, availableData];
+};
+
+export const getBarRadius = (index: number, length: number) => {
+  if (index === 0) {
+    return { bottom: 3 };
+  }
+  if (index === length - 1) {
+    return { top: 3 };
+  }
+  return {};
+};
+
+export const isAvailableBar = (index: number, length: number) => {
+  if (index === length - 1) {
+    return { fill: '#b8bbbe' };
```

Comment:
This should be declared in some variable instead of hard coding the color value here

Reply:
ack